01-city-tours/src/components/header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+
+function Navbar () {
```

Comment:
In React, a major convention is to write components files in PascalCase. So, in this case, should be `Header.jsx` instead of `header.jsx`. The same applies to the other components.

Taking into account that you'll use CSS Modules, the simple folder structure would be:

```
/components
  /Header
    index.js
    Header.jsx
    HeaderStyles.jsx
```

The index.js would only server as a "bridge to import Header.jsx"

```js
import { default } from './Header.jsx'
```
So that in other files you could do:

```js
// import Header from '../components/Header/Header.jsx // no need to do this
import Header from '../components/Header' // you can do a shorter import
```